test/functional/feature_pruning.py

```diff
@@ -127,7 +127,7 @@ def reorg_test(self):
         # Reboot node 1 to clear its mempool (hopefully make the invalidate faster)
         # Lower the block max size so we don't keep mining all our big mempool transactions (from disconnected blocks)
         self.stop_node(1)
-        self.start_node(1, extra_args=["-maxreceivebuffer=20000","-blockmaxweight=20000", "-checkblocks=5"])
+        self.start_node(1, extra_args=["-maxreceivebuffer=20000","-blockmaxsize=5000", "-checkblocks=5"])
```

Comment:
Why this change needs?

Reply:
We removed ```blockmaxweight``` argument from tapyrus. But these tests hadn't been updated. They are some long running scripts listed as ```EXTENDED_SCRIPTS```. These are not run by default by the CI. It has to be enabled in ```configure``` So I did not notice them until now. 
I wanted this change to be a separate PR after checking the running time. But had added them to PR unintentionally.

Reply:
Since multiple changes should not be included in one PR, please delete it from this PR and create new PR.